### PR TITLE
Added "Travel" section to Privacy article

### PR DESCRIPTION
**Pull request by chgangwa**
**Rendered article section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-1480#travel

Requesting a section to be updated in "Edge Privacy White paper" for the users to learn how to the data is managed when the Travel toggle settings is on for them. This content has been approved and signed off by the Privacy team. Kindly approve and publish.

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -410,9 +410,12 @@ The shopping service is turned on by default for all users.  To change the shopp
...
+Microsoft Edge helps you find recommendations while doing your online Travel activities.  To help you find recommendations while planning your travel online, Microsoft Edge downloads a list of travel domains' information locally to the client from the Microsoft Travel service.
+
+When you navigate a website, Microsoft Edge locally determines if the website you're on is a travel domain. If the website is identified as a Travel-related webpage, Microsoft Edge sends the domain, flight dates, from & to locations and passenger count along with information about Microsoft Edge and cookies, if cookies are allowed, to the service. This data does not include any personally identifiable information and is sent over HTTPS. 
+
+The Microsoft Edge travel feature requires sharing of cookie information with Bing. For example, the cookies may be used for debugging, fraud detection, and analytics. When a user visits Bing.com on their browser and updates any settings on Bing pages, Bing.com creates and stores this information in a cookie on user’s browser. This cookie is shared across Bing.com pages, and Edge sends this to the Microsoft Travel service to keep user’s experience consistent.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:418]: @vchapel - curly quotes, &, other fixes needed.  
public version of MWSG but I'd use internal version + Word version: https://docs.microsoft.com/en-us/style-guide/welcome/

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -409,6 +409,19 @@ The shopping service is turned on by default for all users.  To change the shopp
...
+
+When you visit a website, Microsoft Edge locally determines if the website you're on is a travel domain.  If the website is identified as a Travel-related webpage, Microsoft Edge sends the domain, flight dates, From and To locations, and passenger count, along with information about Microsoft Edge and cookies (if cookies are allowed) to the service.  This data does not include any personally identifiable information, and is sent over HTTPS.
+
+The Microsoft Edge Travel feature requires sharing cookie information with Bing.com.  For example, cookies may be used for debugging, fraud detection, and analytics.  When you visit Bing.com in your browser and update any settings on Bing pages, Bing.com creates a cookie in your browser and stores information in the cookie.  This cookie is shared across Bing.com pages, and Microsoft Edge sends this cookie to the Microsoft Travel service to keep your experience consistent.
+
+<!-- todo: revise/delete/update.  'travel' is not found in Settings in stable or in Canary 96 -->
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:420]: @chgangwa - the word 'travel' doesn't appear in Settings.  Is this a future feature?  Delete this paragraph?

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Revise, delete, or comment-out the paragraph about the Travel setting; I'm not seeing such a setting.
Other than that, the Writer/Editor pass is done.

**Review by mikehoffms — APPROVED** [review]

**Comment by mikehoffms** [comment]
I'm keeping @vchapel as a Reviewer just for awareness in light of other in-process PRs.

**Review by vchapel — APPROVED** [review]
LGTM

**Comment by mikehoffms** [comment]
@chgangwa - I moved the Travel section down, in alphabetical order like other sections.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -704,12 +722,8 @@ If WIP is turned on for your version of Microsoft Edge, the browser collects eve
 <!-- ====================================================================== -->
 ## Thank you!
 
-Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open source project.
-
 Microsoft Edge is made possible by the [Chromium][ChromiumMain] open-source project and other open-source software.  To view all of the software credits, navigate to `edge://credits`.  [Google Chrome Privacy Whitepaper][GoogleChromePrivacyWhitepaper] was used as a source for gathering related information about the Chromium open-source project.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:725]: @dalenaMSFT - I removed the dup paragraph.  You pointed it out in a different PR which is currently open.  The dup paragraph came in through a previous PR that's closed/merged to "master" & "live".  The present PR uses the old square brackets convention for URLs (in both copies of the dup paragraph).

Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md]: @dalenaMSFT - please review & approve this PR.

**Review by dalenaMSFT — CHANGES_REQUESTED** [review]
For the travel section, in the instructions to turn off, can we simplify the first item to just the first part (Navigate to `edge://settings/privacy`), and drop the "one way to..." part?

**Comment by mikehoffms** [comment]
> For the travel section, in the instructions to turn off, can we simplify the first item to just the first part (Navigate to `edge://settings/privacy`), and drop the "one way to..." part?

@dalenaMSFT - Fixed.  Please re-review & when ready, approve this PR.

**Review by dalenaMSFT — APPROVED** [review]

**Pull request merged**